sympy/stats/crv.py

```diff
@@ -181,7 +182,9 @@ def _inverse_cdf_expression(self):
         x, z = symbols('x, z', real=True, positive=True, cls=Dummy)
         # Invert CDF
         try:
-            inverse_cdf = solve(self.cdf(x) - z, x)
+            inverse_cdf = solveset(self.cdf(x) - z, x)
+            if isinstance(inverse_cdf, Intersection) and inverse_cdf.args[0] == S.Reals:
```

Comment:
There shouldn't be an error or warning, can you investigate more and let me know?